packages/ui/src/utils/NumberUtil.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Format the given number or string to human readable numbers with the given number of decimals
+ * @param value - The value to format. It could be a number or string. If it's a string, it will be parsed to a float then formatted.
+ * @param decimals - number of decimals after dot
+ * @returns
+ */
+export function formatNumberToLocalString(value: string | number | undefined, decimals = 2) {
```

Comment:
This should be now used under `ui` package: `UiHelperUtil.formatNumberToLocalString`